shared/style/option_menu.css

```diff
@@ -13,3 +13,14 @@ form[role="dialog"][data-subtype="menu"].visible {
   transform: translateY(0);
 }
 
+/* ----------------------------------
+ * Dialogs trick: for further information
```

Comment:
Nit: it's hard to track down the reasoning for this change even with the bug number. I would prefer a more descriptive comment personally.